app/views/cases/overturned_shared/_new_form.html.slim

```diff
@@ -7,36 +7,36 @@ section
   .heading-medium
     = "#{@case.subject} "
     span.no-break
-      = link_to @case.original_case_description,
+      = link_to @case.original_case_description + ' (opens in a new tab)',
               case_path(@case.original_case_id)
 
 hr
 
-.grid-row
-  .column-two-thirds
-    .form-group
-      = f.gov_uk_date_field :received_date, { legend_text: "Date Information Commissioner's Office (ICO) decision was received",
-              form_hint_text: '',
-              today_button: {class: 'button-secondary'} }
-
+fieldset#received_date
+  = f.hidden_field :received_date_dd
+  = f.hidden_field :received_date_mm
+  = f.hidden_field :received_date_yyyy
```

Comment:
why do we need this on the page? Cant we set it in the controllers action?

Reply:
I just wanted to minimize changes and thought that since we handle it this way already ... will remove to clean up the form. As discussed, it doesn't make sense to allow editing of `received_date`.